library/common.h

```diff
@@ -50,4 +50,11 @@
 #define MBEDTLS_STATIC_TESTABLE static
 #endif
 
+/** Allow library to access it's structs' private members.
+ *
+ * Although structs defined in header files are publicly available,
+ * their members are private and should not be accessed by the user.
```

Comment:
just a short note for now, later on when this task is fully implemented, this will need more in-depth explanation